### PR TITLE
Feature/746 vectorize example model

### PR DESCRIPTION
**Pull request by mitzimorris**
#### Submisison Checklist

- [x] Run tests: `./runCmdStanTests.py src/test`
- [x] Declare copyright holder and open-source license: see below

#### Summary:

see issue #746 

#### Intended Effect:

example model reflects best practices, explain choice of prior

#### How to Verify:

```
make build;
make examples/bernoulli/bernoulli
./examples/bernoulli/bernoulli sample data file=examples/bernoulli/bernoulli.data.json
./bin/stansummary output.csv
```

estimate of `theta` should be around 0.2, given data in file `bernoulli.data.json`

#### Side Effects:

N/A

#### Documentation:

N/A

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company): Columbia University

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)

### CODE REVIEW

**Comment by mitzimorris** [comment]
it would be great to get this very minor tweak in before the release!

**Comment by rok-cesnovar** [comment]
Sorry to request a change after approving. Well its actually not a request, just a thought. The cmdstan manual has the non-vector bernoulli model in 2 places. Should we also change those?

There is a footnote stating

```
The model is distributed with Stan’s example models repository, https://github.com/stan-dev/
example-models/blob/master/basic_estimators/bernoulli.stan, and is also available with the
CmdStan distribution at the path examples/bernoulli/bernoulli.stan.
```
I guess just half of that statement will be true now.

**Comment by mitzimorris** [comment]
> Sorry to request a change after approving. Well its actually not a request, just a thought. The cmdstan manual has the non-vector bernoulli model in 2 places. Should we also change those?
> 

yes, absolutely, will clean up manual as well.

File: src/docs/cmdstan-guide/getting-started.tex [hunk 1]
```diff
@@ -181,11 +177,8 @@ \subsection{Data Set}
 the \code{stan\_rdump} in package \code{rstan}.
 %
 \footnote{
-The data is also available from the \code{example-models} repository
-on GitHub,
-\url{https://github.com/stan-dev/example-models/blob/master/basic_estimators/bernoulli.data.R}.
-It is also included with the CmdStan distribution and can be found at path
-\nolinkurl{stan/example-models/basic_estimators/bernoulli.data.R}.
+The data is also included with the CmdStan distribution and can be found at path
+\nolinkurl{stan/example-models/basic_estimators/bernoulli.data.R} and \nolinkurl{stan/example-models/basic_estimators/bernoulli.data.json} .
```
Comment (rok-cesnovar) [inline src/docs/cmdstan-guide/getting-started.tex]: This should just be `\nolinkurl{examples/bernoulli/bernoulli.data.R} and \nolinkurl{examples/bernoulli/bernoulli.data.json}` I think

File: src/docs/cmdstan-guide/stanc.tex [hunk 2]
```diff
@@ -40,7 +40,7 @@ \section{The \stanc Compiler}
 \begin{Verbatim}[fontshape=sl]
 > cd <cmdstan-home>
 > bin/stanc --name=bernoulli --o=bernoulli.hpp \
-  stan/example-models/basic_estimators/bernoulli.stan 
+  example-models/bernoulli/bernoulli.stan
```
Comment (rok-cesnovar) [inline src/docs/cmdstan-guide/stanc.tex]: This should be examples/bernoulli/bernoulli.stan

File: src/docs/cmdstan-guide/stanc.tex [hunk 3]
```diff
@@ -53,7 +53,7 @@ \section{The \stanc Compiler}
 \begin{Verbatim}[fontshape=sl]
 > cd <cmdstan-home>
 > bin\stanc.exe --name=bernoulli --o=bernoulli.hpp ^
-    stan/example-models/basic_estimators/bernoulli.stan 
+    example-models/bernoulli/bernoulli.stan
```
Comment (rok-cesnovar) [inline src/docs/cmdstan-guide/stanc.tex]: And this should also be just examples/bernoulli/bernoulli.stan

**Comment by mitzimorris** [comment]
thanks for the close read - did another pass to clean up this manual.
this time for sure!

**Pull request merged**